src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -295,8 +295,8 @@ internal MatchInfo Clone()
     /// <summary>
```

Comment:
@daxian-dbw, test results were not published for PowerShell-CI-windows at vstfs:///Build/Build/25392